### ISSUE TITLE
Extra host variables not replaced correctly in AndroidManifest.xml

### ISSUE THREAD

**Issue by masimplo** [driftyco/ionic-plugin-deeplinks#64]
Hi I have tried to use the extra host variables but the generated AndroidManifest.xml is wrong.
I set  `$DEEPLINK_HOST` variable to app.domain.com and `DEEPLINK_HOST_2` to demo.domain.com but the generated manifest ignores the _2 part of the variable and replaces just the first part.

i.e:

```
<data android:host="app.domain.com" android:pathPrefix="/" android:scheme="https" />
<data android:host="app.domain.com_2" android:pathPrefix="/_2" android:scheme="https_2" />
<data android:host="app.domain.com_3" android:pathPrefix="/_3" android:scheme="https_3" />
<data android:host="app.domain.com_4" android:pathPrefix="/_4" android:scheme="https_4" />
<data android:host="app.domain.com_5" android:pathPrefix="/_5" android:scheme="https_5" />
```

Note that I have not set variables 3, 4 and 5. I am guessing the variable replacing mechanism is not greedy and stops at the first match, thus replacing all instances of `$DEEPLINK_HOST` with `app.domain.com`

**Comment by adityasaxena** [comment]
@masimplo Thanks for putting this up. Facing the exact same problem. And, even if I remove the extra hosts/schemes from config.xml, when I build, they end up in the AndroidManifest.xml file again. Weird, isnt it ?

**Comment by masimplo** [comment]
@adityasaxena  I believe the fact that AndroidManifest.xml does not update after you remove extra hosts from config.xml is related to #38 
In my case I do a `cordova clean` and `rm -rf platforms` to work around that.

**Comment by adityasaxena** [comment]
@masimplo doesn't work for me - just gave it a shot.

**Comment by adityasaxena** [comment]
@masimplo But, I get what you're saying as well. the variable is replaced as soon as it is matched, not allowed variable substitution to be effective at all.

**Comment by mlynch** [comment]
ah, that's silly. Will fix, thanks.

**Comment by mlynch** [comment]
Fixed and available in 1.0.13. Use `DEEPLINK_2_SCHEME` and so on and so forth
